### DISCUSSION TITLE
Airbyte Setup on windows is failing: fatal: unable to checkout working tree warning: Clone succeeded, but checkout failed.

### DISCUSSION THREAD

**Discussion by nagrotte** [airbytehq/airbyte#34398] [Connector Questions]
Airbyte Setup on windows is failing: fatal: unable to checkout working tree warning: Clone succeeded, but checkout failed.
Is this a known issue ? 
However I tried on Mac , it works fine.

**Comment by marcosmarxm** [comment]
What steps did you made to run Airbyte?

**Comment by nagrotte** [comment]
Steps as described on Airbyte documentation:

https://docs.airbyte.com/deploying-airbyte/local-deployment


Deploy on Windows

After installing the WSL 2 backend and Docker you should be able to run
containers using Windows PowerShell. Additionally, as we note frequently,
you will need docker-compose to build Airbyte from source. The suggested
guide already installs docker-compose on Windows.
Setup Guide

*1. Check out system requirements from Docker documentation
<https://docs.docker.com/desktop/windows/install/>.*

Follow the steps on the system requirements, and necessarily, download and
install the Linux kernel update package.

*2. Install Docker Desktop on Windows.*

Install Docker Desktop <https://docs.docker.com/desktop/windows/install/> from
here.

Make sure to select the options:

   1. *Enable Hyper-V Windows Features*
   2. *Install required Windows components for WSL 2*when prompted. After
   installation, it will require to reboot your computer.

*3. You're done!*

git clone --depth=1 https://github.com/airbytehq/airbyte.git
cd airbyte
bash run-ab-platform.sh


   - In your browser, just visit http://localhost:8000
   - You will be asked for a username and password. By default, that's
   username airbyte and password password. Once you deploy airbyte to your
   servers, be sure to change these.
   - Start moving some data!




Nag Rotte Attitude is Everything!!

**Reply by Alecssander** [reply]
try before: git config --global core.longpaths true

**Comment by seanpden** [comment]
Just wanted to comment here and say I'm running into a similar issue. I'm following these instructions: https://docs.airbyte.com/deploying-airbyte/docker-compose#deploy-on-windows. 

After cloning the repository, I get the following output:
```sh
remote: Enumerating objects: 18689, done.
remote: Counting objects: 100% (18689/18689), done.
remote: Compressing objects: 100% (12433/12433), done.
remote: Total 18689 (delta 7360), reused 11211 (delta 4542), pack-reused 0
Receiving objects: 100% (18689/18689), 118.39 MiB | 2.04 MiB/s, done.
Resolving deltas: 100% (7360/7360), done.
error: invalid path 'airbyte-integrations/connectors/source-klaviyo/.coveragerc '
fatal: unable to checkout working tree
warning: Clone succeeded, but checkout failed.
You can inspect what was checked out with 'git status'
and retry with 'git restore --source=HEAD :/'
```

Attempting ```git restore --source=HEAD``` or ```git switch master``` outputs the following:
```sh
error: invalid path 'airbyte-integrations/connectors/source-klaviyo/.coveragerc '
```

**Comment by adlerdata** [comment]
I followed two instructions to resolve this issue

First I did this:
 - https://stackoverflow.com/questions/63727594/github-git-checkout-returns-error-invalid-path-on-windows
```
"I found that running git config core.protectNTFS false solved my issue, but only if I took some steps before and after running it. The entire process went like so:

1. Clone the Git repository locally;
2. 'cd' into the local Git repository folder that has just been cloned;
3. Run git reset
4. Run git config core.protectNTFS false
5. Run git checkout (just git checkout, no * at the end of command).

After that, I could see the files. Granted, some extra things downloaded from Git that are usually omitted, but it wasn't a big deal in my case."
```

Then, as run-ab-platform.sh didn't work properly and required some web permissions I manually added commands to 
- .env
- docker-compose.yaml 
- flags.yml 

files from below link

 - https://stackoverflow.com/questions/75823212/deploying-airbyte-with-just-docker-compose-yaml-and-varibles

Then I run below docker command.
`docker compose up -d`

[a comment by an automated account is not shown]
